index.html

```diff
@@ -291,8 +291,8 @@ <h2>
           <li>If |state| is {{PermissionState["prompt"]}}, run the following
           steps:
             <ol>
-              <li>If these <a>obtain permission</a> steps were not <a>triggered
-              by user activation</a>, return {{PermissionState["denied"]}}.
+              <li>If the <a>current global object</a> does not have [=transient
```

Comment:
We have to be a bit careful about "current global object" here because transient user activation is defined only for Window.  I think we can instead say:
> If current global object is a Window and it does not have transient activation...

Reply:
Thanks for the quick review. I've added a separate step above to handle the non-Window case and bail out earlier.